x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/operation_support.ts

```diff
@@ -37,16 +37,17 @@ export const getOperationSupportMatrix = (props: Props): OperationSupportMatrix
 
   filteredOperationsByMetadata.forEach(({ operations }) => {
     operations.forEach((operation) => {
+      // replace spread operator by regular push as it's performance wise faster
       if (operation.type === 'field') {
-        supportedOperationsByField[operation.field] = [
```

Comment:
@wylieconlon Replacing push with a spread here happened relatively recently in a refactor PR you did (splitting up the dimension editor). Is there any reason for preferring spreading?